kiele

```diff
@@ -99,6 +99,19 @@ run_vm() {
     "${cmd[@]}"
 }
 
+run_generate_report() {
+    if [[ -f $(dirname $(which kiele))/static-report.html ]]; then
+        report_template=$(dirname $(which kiele))/static-report.html
+    else
+        report_template=$INSTALL_LIB/static-report.html
+    fi
+
+    report_json=${1:-"./report.json"}
+    output_path=${2:-""}
```

Comment:
It's probably a better idea to require the positional parameter instead of have a fallback value, you have it documented that way as well (`<...>` is required, `[...]` is optional). What happens when the user tries to give just the output file here?

How about using the `run_file` variable declared at the bottom of the script instead of declaring `report_json`?